Azure/azure-windows/ingest/parser.yml

```diff
@@ -0,0 +1,764 @@
+name: azure-windows
+pipeline:
+  - name: parse_json
+    description: 'Parse the message'
+    external:
+      name: json.parse-json
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+
+  - name: parse_windows_event
+    description: 'Parse the windows event in the message'
+    external:
+      name: xml.parse-windows-event
+      properties:
+        input_field: '{{parse_json.message.properties.RawXml}}'
+        output_field: message
+
+  - name: extract_provider_guid
+    description: 'Extract the Provider identifier from the GUID'
+    filter: ''
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{parse_json.message.properties.ProviderGuid}}'
+        output_field: result
+        pattern: '{%{UUID:provider_guid}}'
```

Comment:
Do we need a grok there ? It seems there is no data structure, so you could just use a `set`

Reply:
I simplify the way to extract the provider guid from the '{}'